src/data/sql-quiz.ts

```diff
@@ -955,6 +955,16 @@ const sqlQuiz = [
       "'DROP DATABASE <table-name>' drops all tables in the database and deletes the database.",
     Link: "https://dev.mysql.com/doc/refman/8.0/en/drop-database.html"
   },
+  {
+    Question: "In SQL, what does the term 'Relational Division' refer to?",
+    Answer: "A query operation that finds tuples in one relation that match all tuples in another relation",
+    Distractor1: "Dividing a database into smaller relational partitions",
+    Distractor2: "Splitting a relation into different tables based on primary keys",
+    Distractor3: "Dividing numeric data in a SQL query",
+    Explanation:
+      "'Relational Division' is a complex query operation that involves finding all records in one table (relation) that are related to all records in another table, often used for queries like 'find all users who have liked all movies in a category.' This is an advanced operation rarely seen in everyday SQL tasks.",
+    Link: "https://modern-sql.com/use-case/division"
```

Comment:
we only accept links from official docs, MDN, wikipedia, or freeCodeCamp resources like the youtube channel or the news publications.
Please update this to follow the criteria